WebHostLib/templates/userContent.html

```diff
@@ -33,7 +33,18 @@ <h2>Your Rooms</h2>
                             <tr>
                                 <td><a href="{{ url_for("view_seed", seed=room.seed.id) }}">{{ room.seed.id|suuid }}</a></td>
                                 <td><a href="{{ url_for("host_room", room=room.id) }}">{{ room.id|suuid }}</a></td>
-                                <td>{{ room.seed.slots|length }}</td>
+                                <td>
+                                    {% set title = [] %}
+
+                                    {% for slot in room.seed.slots|list|sort(attribute="player_id") %}
```

Comment:
Got around to manually testing this myself, since no one else has really looked at it, but looks like when there's too many it just cuts off (character limit I think) which isn't great.

<img width="134" alt="Screenshot 2024-07-23 at 15 58 19" src="https://github.com/user-attachments/assets/34181c37-a894-4a39-8c5d-a4529363203d">

Also it being a single long list it isn't going to look good with higher numbers of players

However two possible solutions, could split into multiple columns ordered left to right, top to bottom.
<img width="779" alt="Screenshot 2024-07-23 at 22 04 45" src="https://github.com/user-attachments/assets/9a6c4a4a-8f03-4182-84ac-6f6cfe8ff95a">

Or top to bottom left to right.
<img width="778" alt="Screenshot 2024-07-23 at 22 04 21" src="https://github.com/user-attachments/assets/2b9abca2-c3c1-4946-9ac4-9993cfdf648e">

Both require a bit more code for the styling (since you can't style the default tooltip from what I understand) and ordering the slots like this.
This was mocked up in jinja2 but could be processed on the backend.